src/components/Gallery/amp/GalleryContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const GalleryContainer = styled(
+  ({
+    className,
+    ...rest
+  }) => <amp-carousel layout="responsive" type="slides" {...rest} />,
```

Comment:
can move `layout` and `type` into props.

Reply:
It's intentional :). Don't want to give control to developers because this is the recommended way of using gallery. Hence not exposing props.